test/functional/rpc_getorphantxs.py

```diff
@@ -20,6 +20,8 @@ def run_test(self):
         self.wallet = MiniWallet(self.nodes[0])
         self.test_orphan_activity()
         self.test_orphan_details()
+        self.generate(self.wallet, 160)
```

Comment:
if `MAX_ORPHANS` is defined it can be used here directly

Reply:
I've just defined it at the top of the test file should that be fine?